collections/_posts/2023-11-3-StochasticSegmentation.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: review
+title: "Stochastic Segmentation Networks: Modelling Spatially Correlated Aleatoric Uncertainty"
+tags: stochastic segmentation aleatoric uncertainty
+author: "Pierre Rougé"
+cite:
+    authors: "Miguel Monteiro, Loic Le Folgoc, Daniel Coelho de Castro, Nick Pawlowski, Bernardo Marques, Konstantinos Kamnitsas, Mark van der Wilk, Ben Glocker"
+    title:   "Stochastic Segmentation Networks: Modelling Spatially Correlated Aleatoric Uncertainty"
+    venue: NeurIPS 2020
+pdf: "https://arxiv.org/abs/2006.06015"
+---
+
+# Introduction
+
+In segmentation is not obvious what is the correct solution. Especially in medical image segmentation experts can disagree about the objects boundaries.  Ideally this uncertainty about the object boundaries must be capture by the model.
+
+Uncertainty can be divided into two categories:
+
+* Aleatoric uncertainty which is inherent to the observations
+* Epistemic uncertainty which is inherent to the model
+
+In segmentation aleatoric uncertainty vary spatially i.e. an image can have both regions with higher and lower uncertainty.
+
+> *The ideal model should represent the joint probability distribution of the labels at every pixel given the image, enabling sampling multiple plausible label maps*
+
+Aleatoric uncertainty is inherent to the data and thus can not be reduced. So is it of high interest to model it in sensitive applications. Currently deep-learning methods output only one plausible segmentation with no information on uncertainty.  In principle, FCNNs are probabilistic models, since their output is a set of independent categorical distributions per pixel, parameterised by a softmax layer. Because these distributions are independent given the last layer’s activations, sampling from this model would result in spatially incoherent segmentations (grainy label noise in the uncertain regions).
```

Comment:
Order of words:
```suggestion
Aleatoric uncertainty is inherent to the data and thus can not be reduced. So it is of high interest to model it in sensitive applications. Currently deep-learning methods output only one plausible segmentation with no information on uncertainty.  In principle, FCNNs are probabilistic models, since their output is a set of independent categorical distributions per pixel, parameterised by a softmax layer. Because these distributions are independent given the last layer’s activations, sampling from this model would result in spatially incoherent segmentations (grainy label noise in the uncertain regions).
```